### ISSUE TITLE
bug: bind is not a function

### ISSUE THREAD

**Issue by nirgn975** [marcbachmann/opentelemetry-node-metrics#5]
**Describe the bug**
Hi,
I use [nestjs-otel](https://github.com/pragmaticivan/nestjs-otel) latest version (3.0.0) which use this package latest version (1.1.0).
And sometimes (not every build) I got:
```
(node:48789) UnhandledPromiseRejectionWarning: TypeError: meter.createUpDownCounter(...).bind is not a function
    at module.exports (/Users/nir/cybereason/gateway/node_modules/opentelemetry-node-metrics/metrics/version.js:17:6)
    at setupNodeMetrics (/Users/nir/cybereason/gateway/node_modules/opentelemetry-node-metrics/index.js:18:31)
    at OpenTelemetryCoreModule.onApplicationBootstrap (/Users/nir/cybereason/gateway/node_modules/nestjs-otel/src/opentelemetry-core.module.ts:110:44)
```

I just keep saving changes and NestJS build the project, and sometimes I get this error and sometimes I don't (race conduction?)

**To Reproduce**
Just create a new NestJS project, and setup [nestjs-otel](https://github.com/pragmaticivan/nestjs-otel).

**Additional context**
- node.js version: v14.18.0
- npm version: 8.0.0
- macos
- netsjs version 8.x.x
- nestjs-otel version 3.0.0
- opentelemetry-node-metrics version: 1.1.0

**Comment by marcbachmann** [comment]
Weird. Could it be that something gets initialized asynchonously there or in opentelemetry? This library just calls `.getMeter()` https://github.com/marcbachmann/opentelemetry-node-metrics/blob/master/index.js#L6

And uses the `meter.createUpDownCounter` in here
https://github.com/marcbachmann/opentelemetry-node-metrics/blob/master/metrics/version.js#L17

**Comment by weyert** [comment]
The most recent `api-metrics` appears to have removed the `bind`-function. That's probably causing this issue

**Comment by marcbachmann** [comment]
indeed https://github.com/open-telemetry/opentelemetry-js/pull/2559

**Comment by marcbachmann** [comment]
I've merged v2.0.0 that fixes this and dropped compatibility for <v0.27.0
